python/docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'PySpark'
-copyright = u'2014, Author'
+copyright = u''
```

Comment:
If the question is who owns the copyright in the project, no it's not the ASF. Everyone is effectively licensing their contributions to the ASF under the terms of the CLA but don't actually assign copyright. Anyway. For purposes of this line, blank is OK IMHO.